### ISSUE TITLE
Horizontal scrolling on delivery activity modal

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#209]
## Expected Behavior

Modal should be designed to fit the content and avoid unnecessary scroll
## Current Behavior

![image](https://user-images.githubusercontent.com/993369/92313175-f1462a00-ef9e-11ea-8eb1-329c73edc602.png)

![image](https://user-images.githubusercontent.com/993369/92313181-01f6a000-ef9f-11ea-9391-1f41d3fbe6fe.png)

## Environment

https://vm2161.kaj.pouta.csc.fi/

## Possible Solution or Implementation

Material-ui has dynamic width features that help in design  

## Related issues
 
#51
